client/js/lounge.js

```diff
@@ -489,15 +489,6 @@ $(function() {
 	$("#help").on("click", "#view-changelog, #back-to-help", openWindow);
 	$("#changelog").on("click", "#back-to-help", openWindow);
 
-	sidebar.on("click", "#sign-out", function() {
-		socket.emit("sign-out");
-		storage.remove("token");
-
-		if (!socket.connected) {
-			location.reload();
-		}
-	});
-
```

Comment:
I did try to take as much as I could from `lounge.js` 🤷‍♂️
As far as I know, this is the only logic about this in there though.